packages/core/src/appformer/AppFormer.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
+import { Element } from "../core";
+import { Screen } from "./Screen";
+import { Perspective } from "./Perspective";
+
+export class AppFormer {
```

Comment:
Yeah, I'll add these.

Reply:
@tiagobento Thanks! Could you please document methods too?

Also, we could use interface for this class to define contract for this API and do an abstract implementation here. It could help us maintaining backward compatibility in future. Wdyt?

Reply:
Comments added. An interface will be good when we have different implementations, for now, since there's only one, we're sticking with a class.

Reply:
Okay, so does it make sense to have/allow different implementations or not?